### PR TITLE
Replace IRC nickname and channels with Matrix

### PR DESCRIPTION
**Pull request by MichaelKohler**
Just had some fun.. this won't replace any values from the profiles, so most will still point to an IRC nickname ¯\_(ツ)_/¯

### CODE REVIEW

File: remo/settings.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
     'csp.middleware.CSPMiddleware',
     'remo.base.middleware.RegisterMiddleware',
     'waffle.middleware.WaffleMiddleware',
-    'mozilla_django_oidc.middleware.RefreshIDToken',
+    # 'mozilla_django_oidc.middleware.RefreshIDToken',
```
Comment (Mte90) [inline remo/settings.py]: DO you forgotten to uncomment this before to push it?

Comment (MichaelKohler) [reply remo/settings.py]: Yep, totally did not want to do that. Will revert this ASAP. Does the rest look good to you?

**Comment by ioana-chiorean** [comment]
Shouldn't we communicate this? I use different handles for ex. i can add a GitHub ticket for that

**Comment by Mte90** [comment]
Well a communication to the reps is something that will happen when all the changes to the portal will be deployed as usual.
What do you think @MichaelKohler

**Comment by MichaelKohler** [comment]
Sure, done: https://discourse.mozilla.org/t/reps-portal-release-2020-02-04/53295

**Pull request merged**